src/js/control-bar/text-track-controls/text-track-button.js

```diff
@@ -20,6 +20,7 @@ class TextTrackButton extends MenuButton {
   constructor(player, options){
     super(player, options);
 
+    this.update();
```

Comment:
this forces the buttons to update in case the video element has tracks from before we are able to register the handlers to update the buttons.